apps/meteor/app/api/server/v1/im.ts

```diff
@@ -66,13 +66,13 @@ API.v1.addRoute(
 		validateParams: isDmCreateProps,
 	},
 	{
-		post() {
+		async post() {
 			const users =
 				'username' in this.bodyParams
 					? [this.bodyParams.username]
 					: this.bodyParams.usernames.split(',').map((username: string) => username.trim());
 
-			const room = createDirectMessage(users, this.userId, this.bodyParams.excludeSelf);
+			const room = await createDirectMessage(users, this.userId, this.bodyParams.excludeSelf);
```

Comment:
why are adding await here if this message still uses fibers (and .js) ?

Reply:
Huh, I changed createDirectMessage to use new models, but seems merge develop overwrote the file 🤔